lib/core/src/lib/card-view/components/card-view/card-view.component.spec.ts

```diff
@@ -25,22 +25,21 @@ import { TranslateModule } from '@ngx-translate/core';
 import { CardViewSelectItemModel } from '../../models/card-view-selectitem.model';
 import { of } from 'rxjs';
 import { CardViewSelectItemOption } from '../../interfaces/card-view-selectitem-properties.interface';
+import { CardViewItem } from '../../interfaces/card-view-item.interface';
+import { CardViewItemDispatcherComponent } from '../card-view-item-dispatcher/card-view-item-dispatcher.component';
 
 describe('CardViewComponent', () => {
 
     let fixture: ComponentFixture<CardViewComponent>;
     let component: CardViewComponent;
 
-    beforeEach(() => {
+    beforeEach(waitForAsync(() => {
```

Comment:
The purpose of beforeEach(waitForAsync(() => {...})) in the test suite is to handle asynchronous operations that might be involved in the setup phase of the test. Since the compileComponents() method is asynchronous, the waitForAsync function ensures that the test component and its dependencies are fully compiled and ready for testing before proceeding with the actual test cases. I have removed compileComponents as well as waitForAsync

Reply:
As discussed, used async/await instead of waitForAsync, and leave compileComponents